From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Rachapalli Bandi Jagadeesh <jagadeesh.rachapalli.bandi@intel.com>
Date: Thu, 9 Oct 2025 23:19:25 +0530
Subject: [PATCH] Include nospec header for array_index_nospec
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Since array_index_nospec() has been using in xe driver and defined in
nospec.h header file.
For 6.17 current header files including nospec.h.
But for 6.6 LTS, we need to expilcitly include that header.

Signed-off-by: Rachapalli Bandi Jagadeesh <jagadeesh.rachapalli.bandi@intel.com>
---
 drivers/gpu/drm/xe/xe_eu_stall.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/xe/xe_eu_stall.c b/drivers/gpu/drm/xe/xe_eu_stall.c
index af79163..a20421d 100644
--- a/drivers/gpu/drm/xe/xe_eu_stall.c
+++ b/drivers/gpu/drm/xe/xe_eu_stall.c
@@ -7,6 +7,7 @@
 #include <linux/fs.h>
 #include <linux/poll.h>
 #include <linux/types.h>
+#include <linux/nospec.h>
 
 #include <drm/drm_drv.h>
 #include <generated/xe_wa_oob.h>
-- 
2.43.0

